spec/system/admin/enterprise_fees_spec.rb

```diff
@@ -167,42 +167,51 @@
       login_as_admin
       visit admin_enterprise_fees_path
 
-      # edits the first fee
-      select 'Foo', from: "#{prefix}_enterprise_id"
-      select 'Fundraising', from: "#{prefix}_fee_type"
-      fill_in "#{prefix}_name", with: 'Hello!'
-      select 'Inherit From Product', from: "#{prefix}_tax_category_id"
-      select 'Flat Percent', from: "#{prefix}_calculator_type"
+      within fee_row(fee.name) do
+        select 'Foo' # enterprise
+        select 'Fundraising' # fee type
+        find("input[id$='_name']").fill_in with: "Hello!"
+        select 'Inherit From Product' # tax category
+        select 'Flat Percent' # calculator type
+      end
```

Comment:
I'm not 100% confident about this approach. I think it's better than the hard-coded `prefix` and `prefix1` but it would be good to have a better way to identify the select fields. We could add another helper to find the table header label and use the column index to find the right table cell. That would make the comments obsolete.

I didn't want to spend too much time on this though. I hope this version to be helpful as example though.

Reply:
Great minds think alike, I was working on this same type of issue this week.

I started the system spec from the user's point of view and asked myself "how are these fields labelled?". The answer is by the column header; but to ensure the semantic meaning is applied I made use of aria-label (I chose this over `aria-labelled-by` [because it was more practical](https://github.com/openfoodfoundation/openfoodnetwork/commit/ae3cd6f7e046bb1eac5d8e8800bee3dba1f0f7f6)).

Then I was able to create a simple helper based on that: https://github.com/dacook/openfoodnetwork/commit/aff33b79ae24b5bd7d564ea48e64455804eb14bd

Reply:
That's awesome! I like the labeling. It should improve the UX for some people as well. I won't add it to this PR it's good to keep this pattern in mind for next time.